.travis.yml

```diff
@@ -55,4 +54,4 @@ script:
   - bundle exec rails db:structure:load
   - bundle exec rails db:seed
   - rake ci
-  - bundle exec codeclimate-test-reporter
+  - CODECLIMATE_REPO_TOKEN=861a17b4c76ddedf9972cdaca2771f9ab0c99f4ce92298fa965d25a4be55bdbe bundle exec codeclimate-test-reporter
```

Comment:
https://monosnap.com/file/FjaM4YuHoNPU6HSauKaw1YNfOv150u

its 2 different builds, that's what I mean
It is possible to disable the Rails_ENV variable, will see what can i do, I understand and agree that it's ugly